js/state/siteUtil.js

```diff
@@ -444,14 +444,69 @@ module.exports.getDetailFromFrame = function (frame, tag) {
   })
 }
 
-module.exports.getDetailFromTab = function (tab, tag) {
+const getSitesLikeKey = (sites, siteKey, tag) => {
+  if (!sites || !siteKey) {
```

Comment:
sorry my bad for typo. I think it should be `getSitesBySubkey` the plural :stuck_out_tongue: